technical-reports/format/groups.md

```diff
@@ -58,6 +58,16 @@ The names of items in a group are case sensitive. As per the guidance in the [de
 
 ## Additional group properties
 
+<p class="ednote" title="Group properties vs. nested group and token names">
+  The group properties, `$description` and `$type` are prefixed with a dollar sign (`$`). <br/>Future group properties will also use a dollar sign (`$`) prefix. <br/><br/>Group keys without a dollar sign (`$`) prefix denote:
+  <br/>
+  <strong>A token name:</strong> distinguishable by containing a `$value` property
+  <br/>
+  <strong>OR</strong>
+  <br/>
+  <strong>A nested group name:</strong> distinguishable by <em>not</em> having a `$value` property
+</p>
```

Comment:
Consider using a `div` to allow more semantic markup, for example:

```suggestion
<div class="ednote" title="Group properties vs. nested group and token names">

The group properties, `$description` and `$type` are prefixed with a dollar sign (`$`).  
Future group properties will also use a dollar sign (`$`) prefix.

Group keys **without** a dollar sign (`$`) prefix either denote:

- **A token name:** distinguishable by containing a `$value` property
- **A nested group name:** distinguishable by _not_ having a `$value` property

</div>
```

### Preview

<img width="829" alt="Screen Shot 2022-04-25 at 3 02 33 PM" src="https://user-images.githubusercontent.com/85783/165182558-2465aa3a-fa64-49c5-a293-e0b9df74d9e8.png">

Reply:
Why is a div better @kaelig? I would have done the same as Kevin tbh. Maybe I'm missing something though.

Reply:
I think it's so that you can nest paragraphs & lists inside rather than having a single paragraph but visually formatting it over several lines using `<br>`.

FWIW, I second @kaelig's suggestion

Reply:
What @c1rrus said ☝🏻 Thank you!

Reply:
Updated.

Reply:
Closing this - See my newer comment regarding markdown syntax